src/python/WMCore/MicroService/MSManager.py

```diff
@@ -49,6 +50,16 @@ def daemon(func, reqStatus, interval, logger):
         sleep(interval)
 
 
+def daemonOpt(func, interval, logger):
```

Comment:
This is a planned change. It will be worked in this issue [1], when we are about to face the need for variable arguments passed to the daemon caller anyway. 

[1]
https://github.com/dmwm/WMCore/issues/10599